packages/dataviews/src/dataviews-layouts/list/style.scss

```diff
@@ -1,3 +1,7 @@
+ul.dataviews-view-list {
+	list-style-type: none;
```

Comment:
I presume we don't see in our dataviews implementation (pages, patterns, templates) because there's a reset somewhere. Thought we should have this for consumers without having to depend on a different reset.

Reply:
Why not add it below in the existing selector?

Reply:
Not very opinionated. My thinking was that `ul.dataviews-view-list` targets the UL element so it's more intentional.